WooCommerce/Classes/ViewRelated/Inbox/InboxViewModel.swift

```diff
@@ -41,9 +41,12 @@ final class InboxViewModel: ObservableObject {
 
     /// Inbox notes ResultsController.
     private lazy var resultsController: ResultsController<StorageInboxNote> = {
-        let predicate = NSPredicate(format: "siteID == %lld", siteID)
-        let descriptor = NSSortDescriptor(keyPath: \StorageInboxNote.dateCreated, ascending: false)
-        let resultsController = ResultsController<StorageInboxNote>(storageManager: storageManager, matching: predicate, sortedBy: [descriptor])
+        let predicate = NSPredicate(format: "siteID == %lld AND isRemoved == %@", siteID, NSNumber(value: false))
```

Comment:
nit:
```suggestion
        let predicate = NSPredicate(format: "siteID == %lld AND isRemoved == NO", siteID)
```

Reply:
This looks simpler! Updated in https://github.com/woocommerce/woocommerce-ios/pull/6274/commits/5a43691c211a71895a6fd9fa99735fc8d52b6cb8